doc/user_guide/bank/essentials_calendar.md

```diff
@@ -1 +1,34 @@
-Not yet written (We have someone working on it though as of 20240719)
\ No newline at end of file
+# Pick Ups & Deliveries Calendar
+
+The calendar shows the pickup and delivery schedule. It shows past and upcoming distributions in a given month.
+
+Click on the "Pick Ups & Deliveries" section to view the distributions. 
+
+![PickUps & Delivery Calendar](images/pickup&delivery.jpg)
+
+Click on any scheduled distribution to view more details about it.
+The Distribution Schedule page shows details including the Partner,the time of distribution, source inventory, the total number of items, and the status of the distribution.
+
+![Specific Day Distribution](images/specific_day_distribution_schedule.jpg)
+
+If the distribution has been done, you can mark it as complete by clicking on "Distribution Complete". Marking the distribution as complete does not remove it from the calendar.
+
```

Comment:
One good thing about this exercise is that it's prompting future tasks.   *I* think it would be useful for us to have an "exclude completed" filter on this.